plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -262,6 +276,47 @@ def dcnm_get_ip_addr_info(module, sw_elem, ip_sn, hn_sn):
 #             raise Exception(json.dumps(msg_dict))
 
 
+def get_fabric_details(module, fabric):
```

Comment:
Can you add a function doc string for this module to describe what it's doing.  Want to make sure we start doing this for all new functions we add as a best practice.

Reply:
Added doc string